onnx/backend/test/case/node/topk.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+from collections import OrderedDict
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+def top_k_ref(X, k):
+    X_flat = X.reshape((-1, X.shape[-1]))
+    indices_ref = np.ndarray(shape=X_flat.shape, dtype=np.int32)
+    flatten_indices_ref = np.ndarray(shape=X_flat.shape, dtype=np.int32)
+    values_ref = np.ndarray(shape=X_flat.shape, dtype=np.float32)
+    global_idx = 0
+    for i in range(X_flat.shape[0]):
+        od = OrderedDict()
+        for j in range(X_flat.shape[1]):
+            val = X_flat[i, j]
+            if val not in od:
+                od[val] = []
+            od[val].append((j, global_idx))
+            global_idx += 1
+        j = 0
+        for val, idxs in sorted(od.items(), reverse=True):
+            for (idx, flatten_idx) in idxs:
+                indices_ref[i, j] = idx
+                flatten_indices_ref[i, j] = flatten_idx
+                values_ref[i, j] = val
+                j = j + 1
+
+    indices_ref = np.reshape(indices_ref, X.shape)
+    values_ref = np.reshape(values_ref, X.shape)
+
+    indices_ref = indices_ref.take(list(range(k)), axis=-1)
+    values_ref = values_ref.take(list(range(k)), axis=-1)
+
+    return (values_ref, indices_ref)
+
+class TopK(Base):
+
+    @staticmethod
+    def export_top_k():
+        node = onnx.helper.make_node(
+            'TopK',
+            inputs=['x'],
+            outputs=['vales', 'indices'],
+            k=3
+        )
+        x = np.random.randn(3, 4, 5).astype(np.float32)
+        values, indices = top_k_ref(x, 3)
```

Comment:
The top_k_ref function code won't be extracted to the doc together (besides it's pretty long so might not be a good idea to put it into the doc either). Either put a hand-written simple inputs&outputs example here, or use np.sort with slice (you can select a stable sorting algorithm in np.sort).